openapi.yaml

```diff
@@ -1166,7 +1166,60 @@ paths:
       - default
 # {% endif %}
       x-openapi-router-controller: athenian.api.controllers.filter_controller
+  /goal_template/create:
+    post:
+      operationId: create_goal_template
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/GoalTemplateCreateRequest'
+        x-body-name: body
+      responses:
+        200:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CreatedIdentifier'
+          description: Identifier of the created goal template.
+        400:
+          $ref: '#/components/responses/InvalidRequestErrorResponse'
+      security:
+      - bearerAuth: []
+      summary: Create a goal template.
+      tags:
+      - align
+# {% if False %}
+      - default
+# {% endif %}
+      x-openapi-router-controller: athenian.api.controllers.goal_controller
   /goal_template/{id}:
+    delete:
+      operationId: delete_goal_template
+      responses:
+        200:
+          content:
+            application/json:
+              schema:
+                enum:
+                - {}
+                type: object
+          description: Empty object indicates a successful removal.
+        404:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+          description: The goal template was not found.
+      security:
+      - bearerAuth: []
+      summary: Delete a goal template.
+      tags:
+      - align
+# {% if False %}
+      - default
+# {% endif %}
+      x-openapi-router-controller: athenian.api.controllers.goal_controller
```

Comment:
```suggestion
      x-codegen-request-body-name: body
      x-openapi-router-controller: athenian.api.controllers.goal_controller
```
the same below

Reply:
what is the policy here ? only for operations with a body or for every operation ?

Reply:
Every POST/PUT with non-empty body. That's the name of the arguments in the Python function handler. Everything will work without it, actually, but it is added automatically by connexion.